### PR TITLE
[WIP]Implemented colleciton for isodatastore.

### PR DESCRIPTION
**Pull request by anurag03**
Purpose or Intent
===============
Collection conversion of isodatastore.

{{pytest: -v -k 'test_rhev_iso_servicecatalog or test_iso_provision_from_template or test_iso_datastore_crud'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit bb4e1fa4bf4d395a5f97ca5431ef4319bfcf990c:

`cfme/tests/infrastructure/test_iso_provisioning.py`:
- :red_circle: Line 8:1: **F401** *'cfme.infrastructure.pxe.ISODatastoreCollection' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2dddfea666f984c79e530c97a547873d1572797e:

`cfme/tests/services/test_iso_service_catalogs.py`:
- :bangbang: Line 58:101: **E501** *line too long (108 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/infrastructure/test_iso_provisioning.py [hunk 1]
```diff
@@ -52,9 +52,10 @@ def pytest_generate_tests(metafunc):
 
 
 @pytest.fixture
-def datastore_init(iso_cust_template, iso_datastore, provisioning):
+def datastore_init(appliance, iso_cust_template, iso_datastore, provisioning):
     if not iso_datastore.exists():
-        iso_datastore.create()
+        collection = appliance.collections.iso_datastores
+        collection.create(iso_datastore)
```
Comment (anurag03) [inline cfme/tests/infrastructure/test_iso_provisioning.py]: @psav : This is the PR I was talking about, I am not able to understand where iso_datastore value is coming from? and where I have to make changes to make it work.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by anurag03** [comment]
A lot of things in collection has changed so closing this PR.

**Pull request closed without merging**